src/lib/hooks/useQuery.ts

```diff
@@ -33,6 +37,8 @@ export function useLazyQuery<
   variables?: Variables,
   configAndCallbacks?: ConfigAndCallbacks<ReturnedData, Formatter>
 ): UseLazyQueryReturnType<ReturnType<Formatter>, Variables> {
+  const abortControllerRef = useRef<AbortController | null>(null);
```

Comment:
are we not getting abort controller from outside?

Reply:
nope, in case of hook they don't need to pass it, we expose a cancelRequest method which can cancel the latest active request